docs/Account/signers/dynamic.md

```diff
@@ -0,0 +1,151 @@
+---
+sidebar_label: "Dynamic"
+sidebar_position: 2
+---
+
+Dynamic.xyz is a web3 login solution for everyone, offering straightforward onboarding and embedded wallet options for newcomers and elegant and smart login flows for crypto-native users. We aim to abstract away the complexities of user management, offering powerful developer tools around authentication, authorization and onboarding.
+
+## 1. Install Dynamic and Biconomy
+
+In your app's repository, install `@dynamic-labs/sdk-react-core`, `@dynamic-labs/ethereum` and `@biconomy/{account, bundler, common, core-types, paymaster}`:
+
+```bash
+
+yarn add @dynamic-labs/sdk-react-core @biconomy/account @biconomy/bundler @biconomy/common @biconomy/core-types @biconomy/paymaster
+
+```
+
+:::tip
+@dynamic-labs/ethereum will help you to enable all EVM compatible chains including layer 2’s i.e. Base as well as Dynamic Embedded Wallets. Learn more about WalletConnectors [here](https://docs.dynamic.xyz/react-sdk/components/dynamiccontextprovider#walletconnectors).
+:::
+
+## 2. Initialize & configure Dynamic
+
+### 2a. Add the DynamicContextProvider
+
+All you need to get started is [the DynamicContextProvider](https://docs.dynamic.xyz/react-sdk/components/dynamiccontextprovider) - you will want to wrap your app with this component at the highest possible level, like so:
+
+```tsx
+import { DynamicContextProvider } from "@dynamic-labs/sdk-react-core";
+import { EthereumWalletConnectors } from "@dynamic-labs/ethereum";
+import { EthersExtension } from "@dynamic-labs/ethers-v6";
+
+<DynamicContextProvider
+  settings={{
+    // Find your environment id at https://app.dynamic.xyz/dashboard/developer
+    environmentId: "REPLACE-WITH-YOUR-ENVIRONMENT-ID",
+    walletConnectors: [EthereumWalletConnectors],
+    walletConnectorExtensions: [EthersExtension],
+  }}
+>
+  {/* Your app's components */}
+</DynamicContextProvider>;
+```
+
+:::tip
+We are using Ethers here instead of the default Viem, as Biconomy requires an Ethers signer object. Learn more about switching between Viem and Ethers [here](https://docs.dynamic.xyz/react-sdk/viem-ethers).
+
+:::tip
+To enable embedded wallets for your users, toggle it on along with email/social auth [in the dashboard](https://app.dynamic.xyz/dashboard/configurations#emailsocialsignin).
+:::
```

Comment:
Need to break the first tip, this is creating a nested tip
<img width="780" alt="image" src="https://github.com/bcnmy/docs/assets/42104907/2f3fdc51-8d71-4845-8665-d7fb5e2a9c67">